RobotCrawler/src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,18 @@
+package parser;
+
+import org.jsoup.nodes.Document;
+
+import java.util.List;
+
+/**
+ * Functional interface
```

Comment:
Instead of writing comments about functional interface you can use annotation @FunctionalInterface